sanctions/apps/api/v1/views.py

```diff
@@ -88,7 +88,7 @@ def post(self, request):
             metadata = payload.get('metadata', {})
             username = payload.get('username')
             system_identifier = payload.get('system_identifier')
-            sanctions_type = 'ISN,SDN'
```

Comment:
ah, here it is. yeah, we don't want to use a value supplied by the caller here. we only want to use `'ISN,SDN'`

the model itself should have the field be optional, but in this context, we'll always only ever want to use this one value

Reply:
I had originally changed this to not be hard coded since we were getting the `sdn_api_list` value from the payload, and wanted this to reflect the same. But if we don't want that to be an option, makes sense